src/components/CertificateTemplates/tlds/sg/edu/rp/2019-C-SCA/index.js

```diff
@@ -0,0 +1,26 @@
+import PropTypes from "prop-types";
+import MultiCertificateRenderer from "template-utils/MultiCertificateRenderer";
+import { approvedAddresses } from "../common";
+import RPCert from "./certificate";
+
+const templates = [
+  {
+    id: "certificate",
+    label: "Certificate",
+    template: RPCert
+  }
+];
+
+const RPAA2019CSCA = () => (
```

Comment:
Must be

```
const RPAA2019CSCA = props => (
  <MultiCertificateRenderer
    templates={templates}
    whitelist={approvedAddresses}
    {...props}
  />
);
``` 

It will fix integration tests

Reply:
Thank you for pointing out the error.  The code has passed the integration test.